webapp/channels/src/components/sidebar/sidebar_header/browse_or_add_channel_menu.tsx

```diff
@@ -0,0 +1,174 @@
+                    defaultMessage='Open a direct message'
+                />
+            )}
+        />
+    );
+
+    let createUserGroupMenuItem: JSX.Element | null = null;
+    if (props.canCreateCustomGroups) {
+        createUserGroupMenuItem = (
+            <Menu.Item
+                id='createUserGroupMenuItem'
+                onClick={props.onCreateNewUserGroupClick}
+                leadingElement={<AccountMultiplePlusOutlineIcon size={18}/>}
+                labels={(
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.createUserGroupMenuItem.primaryLabel'
+                        defaultMessage='Create new user group'
+                    />
+                )}
+            />
+        );
+    }
+
+    let createNewCategoryMenuItem: JSX.Element | null = null;
+    if (!props.unreadFilterEnabled) {
+        createNewCategoryMenuItem = (
+            <Menu.Item
+                id='createCategoryMenuItem'
+                onClick={props.onCreateNewCategoryClick}
+                leadingElement={<FolderPlusOutlineIcon size={18}/>}
+                labels={(
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.createCategoryMenuItem.primaryLabel'
+                        defaultMessage='Create new category'
+                    />
+                )}
+            />
+        );
+    }
+
+    const invitePeopleMenuItem = (
+        <Menu.Item
+            id='invitePeopleMenuItem'
+            onClick={props.onInvitePeopleClick}
+            leadingElement={<AccountPlusOutlineIcon size={18}/>}
+            labels={(
+                <>
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.invitePeopleMenuItem.primaryLabel'
+                        defaultMessage='Invite people'
+                    />
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.invitePeopleMenuItem.secondaryLabel'
+                        defaultMessage='Add people to the team'
+                    />
+                </>
+            )}
+            trailingElements={props.showInvitePeopleTutorialTip && <InvitePeopleTour/>}
+        />
+    );
+
+    return (
+        <Menu.Container
+            menuButton={{
+                id: ELEMENT_ID_FOR_BROWSE_OR_ADD_CHANNEL_MENU,
+                'aria-label': formatMessage({
+                    id: 'sidebarLeft.browserOrCreateChannelMenuButton.arialLabel',
+                    defaultMessage: 'Browse or create channels',
+                }),
+                class: 'btn btn-icon btn-sm btn-tertiary btn-inverted btn-round',
+                children: <PlusIcon size={18}/>,
+            }}
+            menuButtonTooltip={{
+                text: formatMessage({id: 'sidebarLeft.browserOrCreateChannelMenuButton.tooltip', defaultMessage: 'Browse or create channels'}),
```

Comment:
I don't think we should have a separate translation string for this since the text of this should pretty much always match the label of the button

Reply:
done